pandas/core/frame.py

```diff
@@ -6539,9 +6539,10 @@ def unstack(self, level=-1, fill_value=None):
     See Also
     --------
     %(other)s
-    pivot_table
-    DataFrame.pivot
-    Series.explode
+    pivot_table : Backward fill the new missing values in the resampled data.
```

Comment:
question for @datapythonista 

does this need to be done manually? should be using the first line of the relevant docstring? can this be automated?

Reply:
I think `See Also` can be automated. However I'm not really familiar with pandas's codebase, so I dont know if all `See Also` section can be automated this way (I'm just taking the one line summary from the referenced function).

Reply:
That's a good point.

In some cases it's useful to write the description contextualised for the object being documented. Something like "does the same, but keeping the index", instead of the generic summary.

But worth considering if that bit of extra clarity in the docs is worth having to write every description.

In any case, implementing that is not trivial, since I guess need to be implemented in sphinx. May be we can create an issue for it, and see what other people thing?

Reply:
> In some cases it's useful to write the description contextualised for the object being documented. Something like "does the same, but keeping the index", instead of the generic summary.

ah OK, makes sense.